tests/test_integration_default.py

```diff
@@ -17,9 +17,9 @@ def test_collected(snapshot, actual):
                 assert snapshot == actual
             """
         ),
-        "test_not_collected": (
+        "test_collect": (
```

Comment:
Did this as an easy check because it matches the parameterized test above `test_collected`